### PR TITLE
Store data in nile

### PR DESCRIPTION
**Pull request by gwenshap**
Migrating from a database to Nile entities. 
I kinda love the new look :)

### CODE REVIEW

File: python-flask-todo-list/app/tasks.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "tasks",
+    "schema": {
+      "type": "object",
+      "properties": {
+          "creator": {"type": "string"},
+          "created": {"type": "string"},
+          "task_name": {"type": "string"},
+          "status": {"type": "string"},
+          "due_date": {"type": "string"},
+          "is_private": {"type": "boolean"}
```
[the remark this answers is not in the document]
Comment (MarkRoddy) [reply python-flask-todo-list/app/tasks.json:11]: Have some horror stories I can share privately of people rolling this themselves and not getting the desired results.

[the remark this answers is not in the document]
Comment (MarkRoddy) [reply python-flask-todo-list/app/tasks.json:11]: Just got to the section in your video that mentions the org id enforcement:
applause

[the remark this answers is not in the document]
Comment (MarkRoddy) [reply python-flask-todo-list/app/tasks.json:11]: Just got to the section in your video that mentions the org id enforcement:
applause

**Comment by gwenshap** [comment]
A lot changed and this is no longer valid.

**Pull request closed without merging**